Validation part 1 proposed edits (#125)
jxnl/instructor: docs/blog/posts/validation-part1.md

    messages=[
        {"role": "user", "content": "Extract Jason is 25 years old"},
    ]
    max_retries=3 # (2)!
)

assert user.name == "Jason" # (3)!
assert user.age == 25
```

1.  To simplify your work with OpenAI models and streamline the extraction of Pydantic objects from prompts, we
    offer a patching mechanism for the `ChatCompletion` class.

2.  Invalid responses that fail to be validated succesfully will trigger up to as many reattempts as you define.

> 3.  As long as you pass in a `response_model` parameter to the `ChatCompletion` api call, your response object will

Reviewer: ```suggestion
3.  As long as you pass in a `response_model` parameter to the `ChatCompletion` api call, the returned object will always
```

We can then take advantage of the `model_validator` decorator to perform a validation on a subset of the model's data.

> We're defining a model validator here which runs before `Pydantic` parses the input into its respective fields. That's why we have a **before** keyword used in the `model_validator` class.

```python

> from pydantic import BaseModel,model_validator

Reviewer: ```suggestion
from pydantic import BaseModel, model_validator
```

    [type=value_error, input_value={'chain_of_thought': '1 +... meaning of life is 42'}, input_type=dict]
```

### Validating Citations From Original Text

> Let's see a more concrete example. Let's say that we've asked our model a question about some text source and we want to validate that the generated answer is supported by the source. While we could verify this by looking up the original source manually, a more scalable approach is to use a validator to do this automatically.

Reviewer: ```suggestion
Let's see a more concrete example. Let's say that we've asked our model a question about some text source and we want to validate that the generated answer is supported by the source. This would allow us to minimize hallucinations and prevent statements that are not backed by the original text. While we could verify this by looking up the original source manually, a more scalable approach is to use a validator to do this automatically.
```
